pkg/extensions/sync/utils.go

```diff
@@ -411,29 +413,29 @@ func syncSignatures(client *resty.Client, storeController storage.StoreControlle
 	digests, ok := resp.Header()["Docker-Content-Digest"]
 	if !ok {
 		log.Error().Err(zerr.ErrBadBlobDigest).Str("url", getManifestURL.String()).
-			Msgf("couldn't get digest for manifest: %s:%s", repo, tag)
+			Msgf("couldn't get digest for manifest: %s:%s", remoteRepo, tag)
 
 		return zerr.ErrBadBlobDigest
 	}
 
 	if len(digests) != 1 {
 		log.Error().Err(zerr.ErrBadBlobDigest).Str("url", getManifestURL.String()).
-			Msgf("multiple digests found for: %s:%s", repo, tag)
+			Msgf("multiple digests found for: %s:%s", remoteRepo, tag)
```

Comment:
Can we cover this line by sending empty header in our test request ?

Reply:
use this instead: digest := resp.Header().Get("Docker-Content-Digest")